handlers/main-macos.yml

```diff
@@ -8,20 +8,23 @@
 # to finish and if it's in progress for a longer time, bootstrap fails. We use the old
 # unload/load combo because they actually wait.
 - name: Unload datadog-agent service
-  command: "launchctl unload -wF {{ datadog_macos_system_plist_file_path }}"
```

Comment:
I don't have a strong preference, but I'd probably lean towards leaving the quotes there. Making huge PRs smaller makes them easier to review and it will also make navigating git history easier.